code/modules/reagents/chemistry/readme.md

```diff
@@ -248,6 +248,11 @@ By default, all atom have a reagents var - but its empty. if you want to use an
 			to draw from the contents of this object.
 ```
 
+## FERMICHEM OH GOD OH FUCK
+
+A new and improved way for reagents to react.
+
+
```

Comment:
For future reference, Draft PRs without any substantial content are usually closed immediately - the maintainers may choose not to if they deem, but it's worth noting if this gets closed.

Also, I suggest renaming the system (with Fermi's permission) or simply collapsing it down into being standard methods for chemistry rather than putting a player reference into the codebase - especially one of a player who doesn't even play here often.

Reply:
If it helps, the Fermi part is from Enrico Fermi. Also I don't think people know who I play here..!

Reply:
fermi was a physicist tho